### ISSUE TITLE
Vendors as source account on deposits

### ISSUE THREAD

**Issue by zynexiz** [firefly-iii/firefly-iii#6289]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

When making a withdrawal from an account, you enter the name of the retailer (fx IKEA) as the destination account, and of course your ow account as the source. But when getting money back from the retailer (fx if you return the item), you can't do this. Selecting fx IKEA as source just converts the transaction to withdrawal instead.

As for now, I have to make a separate account (general income or something) that I use as source account. Wouldn't it be better to actually use the name of the vendor instead? This makes the transaction easier to track where it belongs instead of just insert it as a general income. If I look at IKEA account now I don't see any of the deposits made when I get some money back, just see the withdrawals.

### Solution

When making a deposit (fx getting some money back, returned items etc) you should be able to use the retailer name to easier track where the deposit actually belongs. It makes it easier to track, and the balance reflects both withdrawal and deposits made to that account.

### What are alternatives?

Create a separate account for deposits that hold all deposits made for every retailer. Not very convenient.

**Comment by JC5** [comment]
Yes, this is by design. Separate accounts are part of double bookkeeping.

**Comment by zynexiz** [comment]
Ah, have to live with it then :)
